frontends/concrete-python/examples/floating_point/floating_point.ipynb

```diff
@@ -640,9 +640,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": ".venv",
+   "display_name": "Python 3.10.7 64-bit",
```

Comment:
I am not sure for these venv things. I have done as the other notebooks and it works. 
On the CML side, it's managed more cleanly with poetry